var/spack/repos/builtin/packages/anicalculator/package.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+
+from spack.package import *
+
+
+class Anicalculator(Package):
+    """This tool will calculate the bidirectional average nucleotide identity
+    (gANI) and Alignment Fraction (AF) between two genomes.
+
+    Note: A manual download is required for ANIcalculator.
+    Spack will search your current directory for the download file.
+    Alternatively, add this file to a mirror so that Spack can find it.
+    For instructions on how to set up a mirror, see
+    https://spack.readthedocs.io/en/latest/mirrors.html"""
+
+    homepage = "https://ani.jgi.doe.gov/html/download.php?"
+    manual_download = True
+
+    version("1", sha256="236596a9a204cbcad162fc66be3506b2530b1f48f4f84d9647ccec3ca7483a43")
+
+    depends_on("perl@5:")
+
+    conflicts("platform=darwin", msg="ANIcalculator requires Linux")
+
+    def url_for_version(self, version):
+        return "file://{0}/ANIcalculator_v{1}.tgz".format(os.getcwd(), version)
```

Comment:
You can prob do this with a `url`, I don't think `url_for_version` is necessary

Reply:
Do you mean nixing the manual download? I wasn't sure if putting this on spack was a violation of the software license, so I figured the safest option was to have the user manually agree and download.

Reply:
I mean you could probably do something like:
```python
url = "file://{0}/ANIcalculator_v1.tgz".format(os.getcwd())
```
so you don't need to define any function.